README.md

```diff
@@ -268,6 +268,21 @@ $ remme node-account transfer-tokens-from-frozen-to-unfrozen
 }
 ```
 
+Transfer available tokens from unfrozen reputational balance to operational balance (executable only on the machine which runs the node) — `remme node-account transfer-tokens-from-unfrozen-to-operational --amount=1000`.
+
+| Arguments   | Type    | Required | Description                                    |
+| :---------: | :-----: | :------: | ---------------------------------------------- |
+| amount      | Integer | Yes      | Amount to transfer.                            |
```

Comment:
Remove unnecessary blank symbols after `Amount to transfer.`

<img width="682" alt="Screen Shot 2019-05-27 at 11 31 56 AM" src="https://user-images.githubusercontent.com/22666467/58410722-041b4080-8073-11e9-937e-b07995ccb094.png">

Also, fix the arguments column.

<img width="127" alt="Screen Shot 2019-05-27 at 11 32 53 AM" src="https://user-images.githubusercontent.com/22666467/58410780-2b720d80-8073-11e9-87a1-774c963f3c97.png">

Reply:
Done.